src/Dfe.PlanTech.Web/Views/Pages/Page.cshtml

```diff
@@ -20,7 +20,7 @@
     {
         if (Model.Page.Slug == "self-assessment")
         {
-            string sectionName = TempData["SectionName"] as string;
+            string? sectionName = TempData["SectionName"] as string;
```

Comment:
A string is adherantly nullable.  Replace with static typing - using 

var sectionName